exercises/beer-song/canonical-data.json

```diff
@@ -0,0 +1,53 @@
+{
+  "verse": {
+    "description": "return 1 verse",
+    "cases": [
+      {
+        "description": "first verse",
+        "number": 99,
+        "expected": "99 bottles of beer on the wall, 99 bottles of beer.\nTake one down and pass it around, 98 bottles of beer on the wall.\n"
+      },
+      {
+        "description": "middle verse",
+        "number": 44,
+        "expected": "44 bottles of beer on the wall, 44 bottles of beer.\nTake one down and pass it around, 43 bottles of beer on the wall.\n"
+      },
+      {
+        "description": "third to last verse",
+        "number": 2,
+        "expected": "2 bottles of beer on the wall, 2 bottles of beer.\nTake one down and pass it around, 1 bottle of beer on the wall.\n"
+      },
+      {
+        "description": "penultimate verse",
+        "number": 1,
+        "expected": "1 bottle of beer on the wall, 1 bottle of beer.\nTake it down and pass it around, no more bottles of beer on the wall.\n"
+      },
+      {
+        "description": "last verse",
+        "number": 0,
+        "expected": "No more bottles of beer on the wall, no more bottles of beer.\nGo to the store and buy some more, 99 bottles of beer on the wall.\n"
+      }
+    ]
+  },
+  "lyrics": {
+    "#": [
+      "Note:  the input to verses either takes the range of verses to report ",
+      "or nothing.  For the first case, in languages that include a range type ",
+      "structure, tests should use that structure.  In other languages, tests ",
+      "should take 2 values (start and stop) indicating the range boundaries."
+    ],
+    "description": "return multiple verses",
+    "cases": [
+      {
+        "description": "last 4 verses",
+        "start": 3,
+        "stop": 0,
+        "expected": "3 bottles of beer on the wall, 3 bottles of beer.\nTake one down and pass it around, 2 bottles of beer on the wall.\n\n2 bottles of beer on the wall, 2 bottles of beer.\nTake one down and pass it around, 1 bottle of beer on the wall.\n\n1 bottle of beer on the wall, 1 bottle of beer.\nTake it down and pass it around, no more bottles of beer on the wall.\n\nNo more bottles of beer on the wall, no more bottles of beer.\nGo to the store and buy some more, 99 bottles of beer on the wall.\n"
+      },
+      {
+        "description": "all verses",
```

Comment:
Here's an example - maybe not great - but https://github.com/exercism/x-common/blob/master/exercises/hello-world/canonical-data.json
With no name, there is no argument
With name, there is an argument
```
"cases": [
      {
         "description": "no name",
         "expected": "Hello, World!"
      },
      {
         "description": "sample name",
         "name": "Alice",
         "expected": "Hello, Alice!"
      },
   ...
```